### PR TITLE
Update to use ModelingToolkit

### PR DESCRIPTION
**Pull request by tshort**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,92 +87,35 @@ Sims.jl has one main module named `Sims` and the following submodules:
...
+other devices that also return lists of equations. 
 
 The Van Der Pol oscillator is a simple problem with two equations
 and two unknowns:
 
 ``` julia
 function Vanderpol()
-    y = Unknown(1.0, "y")   # The 1.0 is the initial value. "y" is for plotting.
-    x = Unknown("x")        # The initial value is zero if not given.
+    @variables x(t) y(t)
```
Comment (ChrisRackauckas) [inline README.md:103]:
```suggestion
    @variables x(t) y(t) = 1.0
```

can be allowed now that there's the metadata @yingboma?

Comment (tshort) [reply README.md:103]: Interesting. A knew metadata was coming, but I haven't seen examples. Initial values in variables would be great.

Comment (tshort) [reply README.md:103]: Any hints on how to programatically create a variable that has an initial value? I am currently using `Num(Variable{ModelingToolkit.FnType{Tuple{Any},Real}}(gensym("i")))(t)`, and I also see `setmetadata`.

Comment (ChrisRackauckas) [reply README.md:103]: We want to allow interpolating into the macros, i.e . `x = @variables $y = 1.0`

**Pull request merged**